---
layout: page
title: Research
permalink: /research/
---
<img style="float: right;" src="/assets/research_info/HOIs.png" alt="Image of a combinatorial gene regulation" width="200"/>

**Higher-order interactions**

Complex networks are everywhere, and commonly represented as graphs of nodes with interactions. Why graphs? Why restrict interactions to pairwise? Combinatorial and higher-order dependence structures are ubiquitous in Nature, and I'm interested in how to use them to understand complex systems.


<br>
<br>


**Stator: Higher-order genetic interactions and cells states**


I study the role of *higher-order* interactions in particular in biomedicine, and developed a method called [Stator](https://github.com/AJnsm/Stator) to use such higher-order interactions among molecules to identify new cell states hiding in single-cell transcriptomic data sets. [Here](https://www.biorxiv.org/content/10.1101/2023.12.18.572232), we used this to find future fates of embryonic neurons, and liver cancer states that are prognostic of patient survival. 
<img src="/assets/research_info/CC_states.png" alt="Image of a combinatorial gene regulation" width="600"/>


<br>
<br>



**Interactions and information theory**

[I've also shown](https://arxiv.org/abs/2205.04440) how such higher-order interactions are centrally located in the landscape of higher-order information theory. Currently, I'm a postdoc at the <a href="https://www.mis.mpg.de" target="_blank">Max Planck Institute for Mathematics in the Sciences</a>, where I look at the role of higher-order information theory in other complex systems, such as n-qubit quantum circuits. 

<img src="/assets/research_info/commDiag.png" alt="Image of a combinatorial gene regulation" width="600"/>

<br>
<br>

